README.md

```diff
@@ -67,17 +49,25 @@ Below are steps to deploy this Lambda function as a Gooee Smart Home Skill:
             - device:read
             - space:read
             - action:write
+            - user:read
         5. Save.
     * Build Action should provide you with a field to add fulfillment URL. Go back to your lambda function and select the API Gateway trigger. A new section
         should show the API Gateway along with the API endpoint. Copy that endpoint and paste it as the fulfillment URL in the
         Google actions console. Save.
-8. If you do not already have an S3 bucket for the google-assistant app, create one.
-9. In your terminal, in the directory your code is in run commands:
+9. If you do not already have an S3 bucket for the google-assistant app, create one.
+10. In your terminal, in the directory your code is in run commands:
     * `npm install --package-lock`
     * `zip -r ../<NAME_OF_FILE>.zip *`
     * `aws s3 cp ../<NAME_OF_FILE>.zip s3://<BUCKET_NAME> --profile default`
     * `aws lambda update-function-code --function-name <LAMBDA_FUNC_NAME> --s3-bucket <BUCKET_NAME> --s3-key <NAME_OF_FILE>.zip --publish --profile default`
-    * Go back to your AWS Lambda function and in the code section and select "Upload a file from Amazon S3" from the "Code entry type" drop-down. Provide the S3 URL as the upload (this can be found in the **Object URL** section of the bucket object -- make sure you're selecting the object last modified).
-    * Alternatively, you can go to the Lambda console and upload the Zip file from your local system instead of S3.
-10. In the "Handler" type in `lambda_function.fulfillment` or if you've changed the function name it's the module-name.export value in your function. Save.
-11. You can test this action via the Home section in the Google Assistant app on your phone after you hit "TEST" in the Build Action section of the Actions on Google console.
+    * Alternatively, after copying your zip file to s3 you can go back to your AWS Lambda function and in the code section and select "Upload a file from Amazon S3" from the "Code entry type" drop-down. Provide the S3 URL as the upload (this can be found in the **Object URL** section of the bucket object -- make sure you're selecting the object last modified). Or you can go to the Lambda console and upload the Zip file from your local system instead of S3.
```

Comment:
meh

`go back to your AWS Lambda function and in the code section and select`
to
`go back to your AWS Lambda function and in the code section select`

So silly I almost decided against mentioning.

Reply:
```suggestion
    * Alternatively, after copying your zip file to s3 you can go back to your AWS Lambda function and in the code section select "Upload a file from Amazon S3" from the "Code entry type" drop-down. Provide the S3 URL as the upload (this can be found in the **Object URL** section of the bucket object -- make sure you're selecting the object last modified). Or you can go to the Lambda console and upload the Zip file from your local system instead of S3.
```